test/features/send_action_item_to_users_via_email_test.exs

```diff
@@ -17,9 +17,18 @@ defmodule SendActionItemToUsersViaEmailTest do
       retro_path = "/retros/" <> retro.id
       facilitator_session = authenticate(facilitator_session) |> visit(retro_path)
 
+      facilitator_session
```

Comment:
so, the `persist_user_for_retro` helper will run before `persist_idea_for_retro`, [so within the `persist_idea_for_retro` helper](https://github.com/stride-nyc/remote_retro/blob/1eb3cb361295b2c889a119dd23796b260379cd03/test/support/test_helpers.ex#L7-L10), you should be able to grab the persisted user's id and (optionally) use that as the `assignee_id`. I say optionally because we don't want all items to be persisted with an `assignee_id`. only those that come through with a category of `'action-item'`

Reply:
hey @tnewell5. I pulled your branch down, and the idea you created with `persist_idea_for_retro` works just fine with the email! The issue with your test is just that you never updated your assertion. Update it to `assert emails.html_body =~ "Get better (Test User)"` and it blends nicely. You can safely remove lines 20 - 25 and rely on the pre-populated idea, as that will keep the test a bit faster!

Reply:
omg! you're right. I didn't update it because I did not see 'Test user' in the screenshot before but now I do!

Reply:
![image](https://user-images.githubusercontent.com/8727588/34730042-4b1e44ec-f52c-11e7-8d35-d7a34ccaa7bd.png)

Reply:
so somehow the user name is not displayed in the 'action-items' column but is included in the body of the action items emails